### PR TITLE
SageIntacct API:Fyle App Cleanup - Rename, Refactor, Remove, Documentation

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: apps/fyle/connector.py [hunk 1]
```diff
@@ -61,6 +62,24 @@ def _post_request(self, url, body):
 
         elif response.status_code == 500:
             raise InternalServerError('Internal server error', response.text)
+    
+    def __format_updated_at(self, updated_at):
```
Comment (ashwin1111) [inline apps/fyle/connector.py:66]: Type please

File: apps/fyle/tasks.py [hunk 2]
```diff
@@ -45,34 +45,19 @@ def create_expense_groups(workspace_id: int, fund_source: List[str], task_log: T
...
-    }
-    task_log.save()
-
-    return task_log
-
-
-def async_create_expense_groups(workspace_id: int, fund_source: List[str], task_log: TaskLog):
     try:
         with transaction.atomic():
+            updated = []
```
Comment (ashwin1111) [inline apps/fyle/tasks.py:51]: `updated_at`

File: apps/fyle/views.py [hunk 3]
```diff
@@ -335,9 +333,9 @@ def post(self, request, *args, **kwargs):
         )
 
 
-class ExpenseCustomFieldsView(generics.ListCreateAPIView):
+class ExpenseAttributesView(generics.ListAPIView):
```
Comment (ashwin1111) [inline apps/fyle/views.py:336]: Do we need a post for this class?

File: apps/fyle/views.py [hunk 4]
```diff
@@ -440,30 +431,25 @@ class RefreshFyleDimensionView(generics.ListCreateAPIView):
...
-
         except FyleCredential.DoesNotExist:
             return Response(
                 data={
                     'message': 'Fyle credentials not found in workspace'
                 },
                 status=status.HTTP_400_BAD_REQUEST
-            )
-
+            )
```
Comment (ashwin1111) [inline apps/fyle/views.py:455]: new line

**Review by ashwin1111 — CHANGES_REQUESTED** [review]

File: apps/fyle/views.py (hunk 3 above)
Comment (ashwin1111) [reply apps/fyle/views.py]: @NileshPant1999 this isn't fixed yet

Comment (NileshPant1999) [reply apps/fyle/views.py:336]: yes we don't need it

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**